data/org.gnome.shell.gschema.xml.in.in

```diff
@@ -254,6 +254,69 @@ value here is from the GsmPresenceStatus enumeration.</_summary>
       <_summary>Keybinding that shows the force app exit dialog</_summary>
       <_description></_description>
     </key>
+    <key name="activate-first-icon" type="as">
```

Comment:
Was feeling like writing everything out with words when I wrote this. We could also go for `activate-icon-1` or something like that. No particular preference.